addons/l10n_ar_base/__manifest__.py

```diff
@@ -0,0 +1,38 @@
+{
+    'name': 'Argentinian Localization Base',
+    'version': '11.0.1.0.0',
+    'category': 'Localization',
+    'sequence': 14,
+    'author': 'ADHOC SA',
+    'summary': '',
+    'description': """
+Base Module for Argentinian Localization
+========================================
+
+* Activate ARS currency.
+* Add Identification Type model to represent AFIP valid identifications types.
+* Add Countries code defined by AFIP to identify legal entities and natural persons of foreign countries.
+* Add new field named CBU to bank model.
+    """,
+    'depends': [
+        'base',
+        'contacts',
```

Comment:
I think this module only needs base (which do not need to be declared explicitaly)